extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/LDAPGuacamoleProperties.java

```diff
@@ -145,4 +145,17 @@ private LDAPGuacamoleProperties() {}
 
     };
 
+    /**
+     * The maxResult a ldap search can return
```

Comment:
The formatting of this comment is a bit off (asterisks don't line up, capitalization, no period at the end of line).

Also, "The max results" would read better than "the maxResult".